assets/models/organelles/SlimeJet.tscn

```diff
@@ -1,17 +1,14 @@
-[gd_scene load_steps=7 format=2]
+[gd_scene load_steps=6 format=2]
 
 [ext_resource path="res://assets/models/organelles/SlimeJet.glb" type="PackedScene" id=1]
-[ext_resource path="res://shaders/Organelle.shader" type="Shader" id=2]
+[ext_resource path="res://shaders/OrganelleOpaque.shader" type="Shader" id=2]
 [ext_resource path="res://assets/textures/dissolve_noise.tres" type="Texture" id=3]
 [ext_resource path="res://assets/textures/SlimeJet.png" type="Texture" id=4]
-[ext_resource path="res://assets/models/organelles/SlimeJet.anim" type="Animation" id=5]
```

Comment:
What was the slime jet animation for? @Oliveriver ? If the animation is no longer used / it wasn't good, then I think deleting the animation file as well would be good to not confuse people into thinking the animation file does something.

Reply:
when Rose exported the .glb file from blender, the animation was packaged with the mesh so it could just be left overs from implementing that .glb file directly.

Reply:
I can't find any reference to the anim file anywhere else, so if it is actually meant to be used removing it from here will break it entirely.

Reply:
The .glb file has a dependency on the .anim file. Removing the .anim file causes a crash when loading the jet model. I managed to remove the dependency in the .glb file but that meant the animation didn't play. This could be fixed if the animation can be embedded in the model itself, but I don't know how to do that, so for now, we need to leave the .anim file in.

Reply:
The slime jet component *does* actually refer to an animation player, so the changes in this file regarding the removal of the animation need to be reverted. That's the last thing needed in this PR.

Reply:
I've now fixed this myself, and I guess I'll merge this now as I didn't notice that undoing these changes broke anything I could see quickly.